geoAssembler/tests/test_centre_optimiser.py

```diff
@@ -0,0 +1,43 @@
+import geoAssembler.optimiser as centreOptimiser
+import geoAssembler
+
+from extra_data import RunDirectory, stack_detector_data
+from extra_geom import AGIPD_1MGeometry
+
+import numpy as np
+
+import os.path
+
+geom = AGIPD_1MGeometry.from_quad_positions(quad_pos=[
+        (-525, 625),
+        (-550, -10),
+        (520, -160),
+        (542.5, 475),
+    ])
+
+frame_path = os.path.dirname(geoAssembler.__file__) + "/tests/optimiser-test-frame.npy"
+frame = np.load(frame_path)
```

Comment:
Let's move the data loading into the test (or into a fixture, if you want to be fancy), so it doesn't happen on import.

Also, how is it 10 MB? 1 corrected AGIPD frame should be about 4 MB with no compression at all (4 bytes per pixel, 1 megapixel).

Reply:
Moved the import into the function now.

It's 10 MB as I applied an edge mask to the frame, and apparently `np.full((dims...), np.nan)` returns a float64, which I didn't realise, so when it was applied the whole frame became a float64. I use BFG Repo-Cleaner to delete the file and save it as a float32 if you'd like.

Reply:
The proc data is normally float32, so I think it's OK to save it like that. No need for rewriting history at this point, I think, so long as we remember to squash commits when merging the PR.

But can we consider using a stack of module data first - that would also change what we save.